### PR TITLE
Add netcoreapp3.0 bits to forCloudBuild package

### PR DESCRIPTION
**Pull request by kewilli**

### CODE REVIEW

File: Public/Src/Deployment/privatePackages.dsc [hunk 1]
```diff
@@ -60,7 +61,30 @@ namespace PrivatePackages {
                         DetoursServices.Deployment.natives,
                         importFrom("BuildXL.Utilities").withQualifier(net472Qualifier).Branding.brandingManifest
                     ]
-                }
+                },
+                
+                // Net Core App 3.0
```
Comment (erikma) [inline Public/Src/Deployment/privatePackages.dsc]: >// Net Core App 3.0 [](start = 16, length = 19)

Also need netstandard2.0 if available

File: Public/Src/Deployment/privatePackages.dsc [hunk 2]
```diff
@@ -60,7 +61,30 @@ namespace PrivatePackages {
                         DetoursServices.Deployment.natives,
                         importFrom("BuildXL.Utilities").withQualifier(net472Qualifier).Branding.brandingManifest
                     ]
-                }
+                },
+                
+                // Net Core App 3.0
+                Nuget.createAssemblyLayout(importFrom("BuildXL.Engine").withQualifier(winx64Qualifier).Processes.dll),
```
Comment (Kr1sso) [inline Public/Src/Deployment/privatePackages.dsc]: There is tons of duplication here - we'll need to change things around to support pure .netstandard2.0 frameworks only. Please extract the common assemblies into a function that returns them and takes a qualifier as parameter, so you can call it where appropriate.

File: Public/Src/Deployment/privatePackages.dsc (hunk 1 above)
Comment (erikma) [reply Public/Src/Deployment/privatePackages.dsc]: nm, netstandard not available, net472+necoreapp3.0 are fine

File: Public/Src/Deployment/privatePackages.dsc (hunk 2 above)
Comment (erikma) [reply Public/Src/Deployment/privatePackages.dsc]: no need for netstandard after all, but the duplication is a concern.

**Pull request merged**